PyInstaller/hooks/hook-PySide2.QtWebEngineWidgets.py

```diff
@@ -47,7 +47,7 @@ def prefix_with_path(prefix_path, *paths):
         # This is based on the layout of the Mac wheel from PyPi.
         data_path = pyside2_locations['DataPath']
         libraries = ['QtCore', 'QtWebEngineCore', 'QtQuick', 'QtQml',
-                     'QtNetwork', 'QtGui', 'QtWebChannel',
+                     'QtQmlModels', 'QtNetwork', 'QtGui', 'QtWebChannel',
```

Comment:
Nitpick: Please *append* the new string to the line, this makes it easier to see the actual change.

Reply:
I agree, and have initially done so. But then I noticed that the PyQt5 hook already contains the `QtQmlModels` fix, and decided to make the change in the same way as it had been done there. This makes it easier to compare both hooks side-by-side.

Reply:
Your decision was perfectly right then,